liquid_node/configuration.py

```diff
@@ -159,6 +147,25 @@ def load_job(self, name, job_config):
     def app_url(self, name):
         return f'{self.liquid_http_protocol}://{name}.{self.liquid_domain}'
 
+    def _jobs(self):
+        all_jobs = []
+        jobs_to_start = []
+
+        for core_app in Configuration._core_apps:
+            if core_app in Configuration._allways_on_apps and self.ini.has_option('apps', core_app):
+                raise RuntimeError(f'The app "{core_app}" is always on.')
+
+            app_jobs = list(import_string(f'liquid_node.jobs.{core_app}.jobs'))
```

Comment:
Instead of changing the import dance here, I believe it would be cleaner to set a property `app` on all `Job` classes,  so you can keep the `self.jobs` list as it was and build two more lists in the constructor for this class: `self.enabled_jobs = [job for job in self.jobs if self.is_app_enabled(job.app)]` and `self.disabled_jobs = [... if not self.is_app_enabled(job.app)]`. The enabled jobs get run on `deploy`, the disabled ones get stopped on `gc`.